README.md

```diff
@@ -28,21 +30,64 @@ nvm use
 yarn
 ```
 
-### 3. Run MongoDB
+### 3. Configure environment
 
-Prerequisites:
+#### `/.env.template`
 
-- Docker installed
+Copy and rename `.env`.
+
+Set server domain:
+
+```
+SERVER=localhost
+```
+
+Configure database usernames and passwords:
 
-Copy root `.env.template` and rename `.env`. Configure database usernames and passwords:
+```
+MONGO_DB=praise_db
+MONGO_HOST=mongodb
+MONGO_PORT=27017
+MONGO_INITDB_ROOT_USERNAME=praiseDbRootUsername
+MONGO_INITDB_ROOT_PASSWORD=[any password]
+MONGO_USERNAME=praiseDbUsername
+MONGO_PASSWORD=[any password]
+```
+
+#### `/packages/api/.env.template`
+
+Copy and rename `.env`.
+
+Add your Metamask ETH address to `ADMINS` to be able to access Praise dashboard as admin:
 
 ```
-MONGO_INITDB_ROOT_USERNAME=
-MONGO_INITDB_ROOT_PASSWORD=
-MONGO_DB_USER=
-MONGO_DB_PASSWORD=
+ADMINS=0x123..123,0x123..12
 ```
 
+#### `/packages/discord-bot/.env.template`
+
+Copy and rename `.env`.
+
+- `DISCORD_TOKEN` - Your bot's discord token generated via the Discord Developer Portal. You'll need to invite the same bot to your server with the link - `https://discord.com/api/oauth2/authorize?client_id=<client-id>&permissions=378561611840&scope=bot%20applications.commands` (replace `<client-id>` with your bot's client ID), and with the SERVER MEMBERS and MESSAGE CONTENT Intents enabled.
+- `DISCORD_CLIENT_ID` - Your bot's discord client ID, which can be found in the Application settings on Discord Developer Portal
+- `DISCORD_GUILD_ID` - The ID of the server in which you are using the bot. (this can be found by enabling developer mode in Discord, right clicking on te server icon and clicking "Copy Id").
+
+```
+DISCORD_TOKEN=
+DISCORD_CLIENT_ID=
+DISCORD_GUILD_ID=
```

Comment:
I think the discord bot setup may be incomplete -- it keeps crashing for me with the following error:

```
discord-bot-praise | /usr/src/node_modules/discord.js/src/client/websocket/WebSocketManager.js:250
discord-bot-praise |         throw new Error(WSCodes[error.code]);
discord-bot-praise |               ^
discord-bot-praise | Error [DISALLOWED_INTENTS]: Privileged intent provided is not enabled or whitelisted.
discord-bot-praise |     at WebSocketManager.createShards (/usr/src/node_modules/discord.js/src/client/websocket/WebSocketManager.js:250:15)
discord-bot-praise |     at async Client.login (/usr/src/node_modules/discord.js/src/client/Client.js:254:7) {
discord-bot-praise |   [Symbol(code)]: 'DISALLOWED_INTENTS'
discord-bot-praise | }
```

I think the discord bot may also need scopes for `guilds` and `guilds.members.read`?

Reply:
Yes, the discord bot needs additional setup. There is more info in one of the doc pages. The Discord setup should get its own page, I'll make an issue about that.